net.sonic_pi.SonicPi.json

```
@@ -137,26 +143,161 @@
             ]
         },
         {
-            "name": "ruby-ffi",
+            "name": "rtmidi",
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "*.la",
+                "*.a",
+                "*.so"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-5.0.0.tar.gz",
+                    "sha256": "48db0ed58c8c0e207b5d7327a0210b5bcaeb50e26387935d02829239b0f3c2b9"
+                }
+            ]
+        },
+        {
+            "name": "aubio",
             "buildsystem": "simple",
             "build-commands": [
-                "gem install --local ffi-1.13.1.gem"
+                "./waf configure --prefix=$FLATPAK_DEST build",
```

```suggestion
                "./waf configure --prefix=$FLATPAK_DEST build -j $FLATPAK_BUILDER_N_JOBS",
```

so that we can parallel build